feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go

```diff
@@ -218,6 +221,7 @@ func advBGPRouteFromOTG(t *testing.T, args *otgTestArgs, asSeg []uint32) {
 // sent and received IPv4 prefixes.
 func verifyPrefixesTelemetry(t *testing.T, dut *ondatra.DUTDevice, nbr string, wantInstalled, wantSent uint32) {
 	t.Helper()
+	time.Sleep(15 * time.Second)
```

Comment:
Instead of time.Sleep, please use gnmi.Await

Reply:
Actually i tried to address this with gnmi.Await but somehow the timing is not working. With 15seconds timer the tests passes consistently.

Reply:
addressed